src/app/Helpers/ProductHelper.php

```diff
@@ -0,0 +1,11 @@
+<?php
+    if (!function_exists('getNewProduct')) {
+        function getNewProduct($created_at)
+        {
+            $interval = $created_at->diffInDays(\Carbon\Carbon::now());
```

Comment:
Hàm này xử lý gì?

Reply:
Hàm này dùng để tính toán sản phẩm mới đăng, nếu mới đăng 5 ngày sẽ hiển thị là sản phẩm mới.